privileged/Makefile

```diff
@@ -1,10 +1,17 @@
 SOURCE_FILES := $(shell test -e src/ && find src -type f)
 
 policy.wasm: $(SOURCE_FILES) Cargo.*
+	# rustup target add wasm32-wasi 
+	# rustup target add wasm32-unknown-unknown
 	# cargo build --target=wasm32-wasi --release
 	# cp target/wasm32-wasi/release/*.wasm policy.wasm
-	cargo build --release --target wasm32-unknown-unknown
-	cp target/wasm32-unknown-unknown/release/*.wasm policy.wasm
+	cargo build --release --target wasm32-wasi
+	cp target/wasm32-wasi/release/*.wasm policy.wasm
+
+.PHONY: run
+run:
+	go get github.com/tetratelabs/wazer
```

Comment:
do we need this?

Reply:
that should be wazero. we need it for host.go